support-frontend/assets/helpers/paymentIntegrations/readerRevenueApis.js

```diff
@@ -40,27 +40,37 @@ export type StripePaymentMethod = 'StripeCheckout' | 'StripeApplePay' | 'StripeP
 export type StripePaymentRequestButtonMethod = 'none' | StripePaymentMethod;
 
 type RegularContribution = {|
+  productType: 'Contribution',
   amount: number,
   currency: string,
   billingPeriod: BillingPeriod,
 |};
 
 export type DigitalSubscription = {|
+  productType: 'DigitalSubscription',
   currency: string,
   billingPeriod: BillingPeriod,
   readerType: ReaderType,
-  giftMessage?: string | null,
-  giftDeliveryDate?: string | null,
 |};
```

Comment:
Why remove  `giftMessage` and `giftDeliveryDate` from the `DigitalSubscription` type?

Reply:
I think our comments crossed over - these are not used on the server side and I think were left over from a prototype.